readme.md

```diff
@@ -56,10 +56,10 @@ or
 osc-cli --service SERVICE --call CALL [PROFILE] [--CALL_PARAMS ...]
 ```
 with 
-* SERVICE one of the services provided by Outscale (fcu, lbu, eim, directconnect, icu)
-* CALL the call you request (ie DescribeInstances)
-* PROFILE the profile you want to connect to (optional)
-* CALL_PARAMS [call arguments](http://docs.outscale.com) which are case-sensitive (optional)
+* SERVICE, one of the services provided by Outscale (fcu, lbu, eim, directlink, icu)
+* CALL, the call you request (ie DescribeInstances)
```

Comment:
okms is missing, isn't it ?

Reply:
We will add OKMS here when all the OKMS related documentation is public.

Reply:
OK